Makefile

```diff
@@ -734,7 +734,11 @@ OPTIMIZE    = -O0
 LTO_FLAGS   = $(OPTIMIZE)
 else
 OPTIMIZE    = -Os
+ifeq ($(TARGET),$(filter $(TARGET),$(F1_TARGETS)))
```

Comment:
@borisbstyle try enabling LTO but compiling using gcc-4.9.3. In INAV I've had some pretty nasty and unexplained bugs that were resolved by downgrading the compiler OR disabling optimisations.